projects/fluxcd/flux2/Makefile

```diff
@@ -12,14 +16,28 @@ EXTRA_GO_LDFLAGS=-X main.VERSION=$(GIT_TAG:v%=%)
 FLUX_MANIFESTS_TARGET=$(REPO)/cmd/flux/manifests
 
 BINARY_PLATFORMS=linux/amd64 linux/arm64 darwin/amd64 darwin/arm64
+FETCH_BINARIES_TARGETS=eksd/kubernetes/client
+
+BUILD_TARGETS=local-images
```

Comment:
I don't think we want this, it will override our default targets we try to run for all projects. Was there a target that wouldn't succeed for some reason without this?